src/pages/Admin/Charity/Dashboard/Dashboard.tsx

```diff
@@ -32,44 +32,57 @@ export default function Dashboard() {
         }) => {
           const earningsPayoutsPending = payoutsPending - donationsBal;
           const earningsPayoutsMade = totalEarnings - earningsPayoutsPending;
+          const donationsPaidOut =
```

Comment:
I'm trying to understand which of these 2 fields was causing the final value to be `NaN`, did you maybe figure that out?

Since the problematic fields were `paidOut` in both boxes (donation and sustainability fund), i.e.:
- `paidOut={payoutsMade - earningsPayoutsMade}`
- `paidOut={earningsPayoutsMade}`

AND `earningsPayoutsMade = totalEarnings - earningsPayoutsPending`
BUT `earningsPayoutsPending` displays just fine in field `Sustainability Fund > pending`

It's safe to say that the possible culprits are `totalEarnings` and `payoutsMade`. Maybe AWS is not returning valid number values for these 2 fields? :thinking:

Reply:
@misicnenad It was totalEarnings at fault. Cleaned up the other in 198786d6a.